views/songs/language.pug

```diff
@@ -1,20 +1,38 @@
 extends ../layout
 
 block content
+    -var permit = true
+
     h2 Available Language:
+    if success_messages[0]
+        .alert.alert-success
+            each success_message in success_messages
+                p #{success_message}
+    if error_messages[0]
+        .alert.alert-danger
+            each error_msg in error_messages
+                p #{error_msg}  
+
+    script(type="text/javascript").
+        $(document).ready(function() {
+            setTimeout(function() {
+                $(".alert").alert('close');
+            }, 3000);
+        });
+
     ol
         each language in languages
             li(id=`${language.code}`)
               span.display
-                a(href="#" onclick=`editLanguage('${language.code}')`) #{language.label}
+                a(href="#" onclick=`editLanguage('${languages.length}','${language.code}')`) #{language.label}
```

Comment:
why is length being passed in now?

Reply:
I forgot to delete it after using 'permit' variable to handle blocking.